common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -387,10 +390,7 @@ public void onReceive(@NonNull PropertyBag propertyBag) {
             final Bundle resultBundle = mBrokerResultFuture.get();
 
             final String negotiatedBrokerProtocolVersion = interactiveRequestIntent.getStringExtra(NEGOTIATED_BP_VERSION_KEY);
-            // For MSA Accounts Broker doesn't save the accounts, instead it just passes the result along,
```

Comment:
nit: why removing this?

Reply:
I think this is an old comment, because Broker does save the MSA accounts. This detail is actually important to my implementation because we query broker storage for msa accounts.

Reply:
@mohitc1 Let me know if i'm understanding this wrong. I can add bnack the comment if it's still relevant.

Reply:
I've returned the comment, we can remove it later if necessary.